### DISCUSSION TITLE
Fail to load woff2 typeface

### DISCUSSION THREAD

**Discussion by namse** [rust-skia/rust-skia#917] [Q&A]
Is it disabled to load woff2?
For example, skia's canvaskit turns on `skia_use_freetype_woff2` build parameter (https://github.com/google/skia/blob/b7ac9da3e5ac34baf5da70b3e37cdbfd898ed54b/modules/canvaskit/compile.sh#L113)

Is there any option to turn on that?

**Comment by pragmatrix** [comment]
On which platform were you expecting woff2 support?

By default, Skia uses the system's FreeType library on Linux based platforms, so there is no control whether if FreeType supports woff2 typefaces (a prerequisite for that is the brotli library).

We may consider supporting woff2 when the embed-freetype feature is enabled, which would would add about 200kb to the Skia library and therefore deserves a new feature flag. In addition to that, FreeType may be embedded on other platforms, like on macOS and Windows, but I am not sure yet if FreeType is actually used if enabled.

**Comment by namse** [comment]
I'm using skia to make games, and currently my main platform is x86_64-pc-windows-msvc. We are thinking of supporting ios, android, free-bsd, etc. in the future.

Recently, I found out that woff2's size is small because it uses brotli. So, if it is difficult for rust-skia to provide woff2 by default, I think it would be okay to use an external library directly, convert woff2 to ttf, etc. and load it into skia.
